### PR TITLE
Update the developer documentation for the e2e tests.

### PR DESCRIPTION
**Pull request by utam0k**

### CODE REVIEW

File: docs/src/developer/e2e/runtimetest.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+./youki: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=...., for GNU/Linux 3.2.0, with debug_info, not stripped`
+```
+
+This output indicates that the binary is dynamically linked, thus cannot be run inside the container process
+
+```console
+./runtimetest: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), statically linked, BuildID[sha1]=...., for GNU/Linux 3.2.0, with debug_info, not stripped
+```
+
+This output indicates that the binary is statically linked, and can be run inside the container process
```
Comment (YJDoc2) [inline docs/src/developer/e2e/runtimetest.md:44]: Can we have the lines `This output indicates...` before their respective console examples? This way can be a bit confusing.

**Review by YJDoc2 — APPROVED** [review]
overall lgtm, a minor nitpick

**Comment by YJDoc2** [comment]
@utam0k may I ask you to do the naming changes for `oci-tests` -> `test-oci` and similar, that are done after https://github.com/containers/youki/pull/2330 merge?

File: docs/src/developer/e2e/runtimetest.md (hunk 1 above)
Comment (utam0k) [reply docs/src/developer/e2e/runtimetest.md:44]: Thanks for your comment, But it is only copied from the original. We have to take care of it in another PR.
https://containers.github.io/youki/developer/runtimetest.html

Comment (YJDoc2) [reply docs/src/developer/e2e/runtimetest.md:44]: ohk, we can do it separately then :+1:

**Pull request merged**